scripts/run_regtest.sh

```diff
@@ -60,12 +60,7 @@ docker run \
        -e DAEMON_URL="dbb:dbb@${DOCKER_IP}:10332" \
        -e COIN=BitcoinSegwit \
        -e NET=regtest \
-       -e RPC_PORT=10002 \
```

Comment:
don't just delete, but replace with the new way of specifying those?

https://github.com/kyuupichan/electrumx/blob/master/docs/environment.rst#services